src/pattern-library/components/patterns/input/RadioGroupPage.tsx

```diff
@@ -0,0 +1,189 @@
+import Library from '../../Library';
+
+export default function RadioGroupPage() {
+  return (
+    <Library.Page
+      title="Radio group"
+      intro={
+        <p>
+          <code>RadioGroup</code> is a <code>radiogroup</code> composite
+          component that includes a list of <code>radio</code>s in it.
+        </p>
+      }
+    >
+      <Library.Pattern>
+        <Library.Usage componentName="RadioGroup" />
+        <Library.Example>
+          <Library.Demo
+            title="Basic RadioGroup"
+            withSource
+            exampleFile="radio-group-horizontal"
+          />
+        </Library.Example>
+      </Library.Pattern>
+
+      <Library.Pattern title="Working with RadioGroups">
+        <p>
+          <code>RadioGroup</code> can render a list of radios arranged
+          horizontally or vertically. Each radio can optionally render a
+          subtitle, and be individually disabled.
+        </p>
+        <p>
+          Radios can be focused via arrow keys. Right/Left for horizontal{' '}
+          <code>RadioGroup</code>s, and Up/Down for vertical ones.
+        </p>
+
+        <Library.Example title="RadioGroup direction">
+          <Library.Demo
+            title="Vertical RadioGroup"
+            withSource
+            exampleFile="radio-group-vertical"
+          />
+        </Library.Example>
+
+        <Library.Example title="Labelling RadioGroups">
+          <p>
+            There are two ways to label a <code>RadioGroup</code>. Make sure to
+            use at least one of them.
+          </p>
+
+          <Library.Demo
+            title="Via aria-label"
+            withSource
+            exampleFile="radio-group-aria-label"
+          />
+          <Library.Demo
+            title="Via aria-labelledby"
+            withSource
+            exampleFile="radio-group-aria-labelledby"
+          />
+        </Library.Example>
+      </Library.Pattern>
+
+      <Library.Pattern title="Component API">
+        <Library.Example title="inputs">
```

Comment:
The logic behind the ordering of the props here is not apparent. I think we should keep it simple and list everything in alphabetical order followed by the `htmlAttributes` rest prop.

Reply:
Right, I followed an almost random order 😅